packages/ckeditor5-list/docs/_snippets/features/todo-list.js

```diff
@@ -32,6 +32,9 @@ ClassicEditor
 				'toggleImageCaption', 'imageTextAlternative', 'ckboxImageEdit'
 			]
 		},
+		ckbox: {
+			forceDemoLabel: true
+		},
```

Comment:
For some reason I don't see `insert with file manager` button in this demo.
![image](https://github.com/ckeditor/ckeditor5/assets/95343030/1531707f-cf16-468e-b83f-170d31b90db4)

Reply:
@niegowski - can you please have a look at it in your spare time? The imports are there, just don't work.  
  
I don't find this a bloker for merging.

Reply:
Further observations: looks like CKBox is imported, but does not work anyway - adding it to the toolbar returned an error: 

![](https://github.com/ckeditor/ckeditor5/assets/68123541/effe0b5b-76ba-4b16-8a36-696623ca0e97)

Reply:
Some problems with the editing, too. I comment the service out until we can identify and fix the problem.